integration/test/Testlib/Env.hs

```diff
@@ -101,7 +101,18 @@ data HostPort = HostPort
 
 instance FromJSON HostPort
 
-data Service = Brig | Galley | Cannon | Gundeck | Cargohold | Nginz | Spar | BackgroundWorker | Stern
+data Service
+  = Brig
+  | Galley
+  | Cannon
+  | Gundeck
+  | Cargohold
+  | Nginz
+  | Spar
+  | BackgroundWorker
+  | Stern
+  -- Don't add federator here. Federator is problematic because it has two http servers
+  -- and this startup design assumes that all services have one and only one host+port.
```

Comment:
Can you please not change anything here, I have found a way to add federator stuff here.